qa/tasks/cephfs/test_quota.py

```diff
@@ -104,3 +104,69 @@ def test_remote_update_write(self):
         with self.assertRaises(CommandFailedError):
             self.mount_b.write_n_mb("subdir_data/file", 40)
 
+    def test_human_readable_quota_values(self):
+        self.mount_a.run_shell(["mkdir", "subdir"])
+
+        self.assertEqual(self.mount_a.getfattr("./subdir",
+                                               "ceph.quota.max_bytes"), None)
+
+        # kibibyte
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "10K")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "10240")
```

Comment:
nit: There's lot of code repetition here onwards. Perhaps, it would be easier to read this if you use a loop -

```
sizes = ['10K', '10240', '100Ki', '10M', ...]
for size in sizes:
    self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", size)
```

Reply:
this depends on what we conclude in https://github.com/ceph/ceph/pull/49008#discussion_r1068034562, if we think we will allow only `(K/G/T/)i` there then a lot of code from the testcase will need to be removed